.travis/test_connection_to_torodb.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -ev
+
+# Wait until torodb it's ready (or timeout after 10s)
+COUNTER=10
+while ! nc -z localhost 27019 && [ $COUNTER -gt 0 ]
```

Comment:
Checking the output of the Travis build it seems that this bit is working and the port `27019` is up&running. I don't get why it's failing as if the port wasn't ready later on.

It also doesn't show any "Waiting..." line, is it possible that I am not reading the output properly? Becauase as per code this should be working:

1) it waits
2) it doesn't timeout because it doesn't show any waiting line
3) it tries to connect to a port that we already know it's open